Gazebo/models/wsc_aircraft/model.sdf

```diff
@@ -0,0 +1,1803 @@
+    </joint>
+
+    <joint name="battery_joint" type="fixed">
+      <pose>0 -1.3254 0.795 0 0 0</pose>
+      <parent>fuselage</parent>
+      <child>battery</child>
+    </joint>
+
+    <joint name="motor1_joint" type="fixed">
+      <pose relative_to='motor1'/>
+      <parent>fuselage</parent>
+      <child>motor1</child>
+    </joint>
+
+    <joint name="motor2_joint" type="fixed">
+      <pose relative_to='motor2'/>
+      <parent>fuselage</parent>
+      <child>motor2</child>
+    </joint>
+
+    <joint name="propeller1_joint" type="revolute">
+      <pose relative_to='propeller1'/>
+      <parent>motor1</parent>
+      <child>propeller1</child>
+      <axis>
+        <xyz>0 0 1</xyz>
+        <dynamics>
+          <damping>0.2</damping>
+          <friction>2</friction>
+        </dynamics>
+      </axis>   
+      <sensor name="prop1_sensor" type="force_torque">
+        <always_on>true</always_on>
+        <update_rate>60</update_rate>
+        <visualize>true</visualize>
+        <topic>prop1_force_joint/force_torque</topic>
+        <force_torque>
+          <frame>sensor</frame>
+          <measure_direction>parent_to_child</measure_direction>
+        </force_torque>
+      </sensor>   
+    </joint>
+
+    <joint name="propeller2_joint" type="revolute">
+      <pose relative_to='propeller2'/>
+      <parent>motor2</parent>
+      <child>propeller2</child>
+      <axis>
+        <xyz>0 0 -1</xyz>
+        <dynamics>
+          <damping>0.2</damping>
+          <friction>2</friction>
+        </dynamics>
+      </axis>   
+      <sensor name="prop2_sensor" type="force_torque">
+        <always_on>true</always_on>
+        <update_rate>60</update_rate>
+        <visualize>true</visualize>
+        <topic>prop2_force_joint/force_torque</topic>
+        <force_torque>
+          <frame>sensor</frame>
+          <measure_direction>parent_to_child</measure_direction>
+        </force_torque>
+      </sensor>     
+    </joint>
+
+    <!-- Sensors -->
+
+    <link name='imu_link1'>
+      <inertial>
+        <pose>0 0 0 0 0 0</pose>
+        <mass>0.11</mass>
+        <inertia>
+          <ixx>0.00002</ixx>
+          <ixy>0</ixy>
+          <ixz>0</ixz>
+          <iyy>0.00002</iyy>
+          <iyz>0</iyz>
+          <izz>0.00002</izz>
+        </inertia>
+      </inertial>
+      <sensor name="imu_sensor1" type="imu">
+        <pose>0 0 0 3.141593 0 -1.5707963</pose>
+        <always_on>1</always_on>
+        <update_rate>1000.0</update_rate>
+      </sensor>
+    </link>
+    <joint name='imu_joint1' type='revolute'>
+      <child>imu_link1</child>
+      <parent>wing_body</parent>
+      <axis>
+        <xyz>0 0 1</xyz>
+        <limit>
+          <lower>0</lower>
+          <upper>0</upper>
+          <effort>0</effort>
+          <velocity>0</velocity>
+        </limit>
+        <dynamics>
+          <damping>1.0</damping>
+        </dynamics>
+      </axis>
+    </joint>
+
+<!-- Gimbal camera links and joints  -->

```

Comment:
There is a `gimbal_3d_small` model available in `ardupilot_gazebo` that could be used here with an `<include>` block.